refactor/results/app/api/api_v0/endpoints/workflow.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+import io
+import logging
+import logging.config
+import os
+import random
+import sqlite3
+import string
+import time
+from os import path
+from tempfile import TemporaryFile
+from typing import Any, BinaryIO, Optional, Tuple, Union
+
+import cloudpickle as pickle
+import requests
+from app.schemas.common import HTTPExceptionSchema
+from app.schemas.workflow import InsertResultResponse, Node, Result, UpdateResultResponse
+from fastapi import APIRouter, HTTPException, Request, UploadFile
+from fastapi.responses import FileResponse, StreamingResponse
+
+logging.config.fileConfig("logging.conf", disable_existing_loggers=False)
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+fs_server_address = os.environ.get("FS_SERVER_ADDRESS")
```

Comment:
this should probably call `get_config` and reference the config file.  You could add something like this to this file:

```
_WORKFLOW_DEFAULTS = {
    "service": {
        "fs": {
            "address": os.environ.get("COVALENT_FS_SERVER_ADDRESS") or "localhost",
            "port": int(os.environ.get("COVALENT_FS_PORT")) or 8000,
        }
    }
}

update_config(_WORKFLOW_DEFAULTS, override_existing=False)
```

which would add the following entry to the config, if it does not exist:
```
[service.fs]
address = "localhost"
port = 8000
```

Reply:
I think after the supervisord work is merged into `develop` we can simply use environment variables as the `supervisord.conf` (supervisord config file) will manage the environment variables for all the services

Reply:
Great, disregard my comment here then.

Reply:
So is the consensus here that env variables are the way to go?